polkadot/scripts/build-only-wasm.sh

```diff
@@ -27,11 +23,17 @@ else
   export WASM_TARGET_DIRECTORY=$2
 fi
 
+
+
 if [ -d $WASM_BUILDER_RUNNER ]; then
   export DEBUG=false
   export OUT_DIR="$PROJECT_ROOT/target/release/build"
-  fl_cargo run --release --manifest-path="$WASM_BUILDER_RUNNER/Cargo.toml" \
+  fl_cargo run --release --manifest-path="$WASM_BUILDER_RUNNER/Cargo.toml" --features fast-runtime \
     | grep -vE "cargo:rerun-if-|Executing build command"
 else
-  fl_cargo build --release -p $1
+  fl_cargo build --release -p $1 --features fast-runtime
```

Comment:
This script is generic. If you want to be able to pass `--features`, you should make it possible from the outside to pass arguments.

Reply:
Yeah, this was not ready for review, forgot to mark it as a draft.